packages/react/src/List/OrderedList.stories.tsx

```diff
@@ -0,0 +1,27 @@
+import {ComponentMeta, ComponentStory} from '@storybook/react'
+import React from 'react'
+import {OrderedList} from '.'
+
+export default {
+  title: 'Components/OrderedList',
+  component: OrderedList,
+  argTypes: {
+    children: {
+      table: {
+        disable: true
```

Comment:
Nice to have for future, we should re-enable this so folks can change the text, especially in Playground. Here's how I [did it for AnchorNav](https://primer.style/brand/storybook/?path=/story/components-anchornav--playground).

Reply:
Thats an interesting approach. My only concern is that it would muddle the story by adding data and the context how you have done it which might confuse a user

`({data, ...args}: {data: MockData; offset: number}) => {`

This is in the beginning of the story in Storybook